sourcemash/api/users.py

```diff
@@ -50,12 +49,14 @@ def __init__(self):
     def get(self):
         return { 'user': marshal(current_user, user_fields) }
```

Comment:
That was always true, which is why we started with the whole .get('email') thing.